pico_sdk_version.cmake

```diff
@@ -3,10 +3,18 @@
 set(PICO_SDK_VERSION_MAJOR 1)
 # PICO_BUILD_DEFINE: PICO_SDK_VERSION_MINOR, SDK minor version number, type=int, pico_base
 # PICO_CONFIG: PICO_SDK_VERSION_MINOR, SDK minor version number, type=int, pico_base
-set(PICO_SDK_VERSION_MINOR 1)
+set(PICO_SDK_VERSION_MINOR 2)
 # PICO_BUILD_DEFINE: PICO_SDK_VERSION_REVISION, SDK version revision, type=int, pico_base
 # PICO_CONFIG: PICO_SDK_VERSION_REVISION, SDK version revision, type=int, pico_base
 set(PICO_SDK_VERSION_REVISION 0)
+# PICO_BUILD_DEFINE: PICO_SDK_VERSION_PRE_RELEASE_ID, optional SDK pre-release version identifier, type=string, pico_base
+# PICO_CONFIG: PICO_SDK_VERSION_PRE_RELEASE_ID, SDK version revision, type=string, pico_base
```

Comment:
Oh, I guess the PICO_CONFIG description also needs editing to match the PICO_BUILD_DEFINE description? "optional SDK pre-release version identifier"

And I guess I need to update `extract_configs.py` to also look for `PICO_CONFIG` entries in `*.cmake` files too :grin: (currently it only looks in `*.c` and `*.h` files)

...and for all the "config items" in this file, it looks like `, pico_base` needs correcting to `, group=pico_base`.

Reply:
ah yes; this seemed like a better place for them to be in this case